notebooks/abbash_notebook.ipynb

```diff
@@ -1,28 +1,540 @@
 {
```

Comment:
<p>more samples = smaller step, and peak is sharper. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>